dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,42 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.lang.Double;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
```

Comment:
I realized we're missing to test the main thing here. What was the purpose of our transformation - to make the object exactly java.math.BigDecimal (not some child of it). Where do we check that?

Which makes me thing - what if we name it `BigDecimalExactClassTransformer` or `BigDecimalToExactClassTransformer`?